model/testing/src/icon4py/model/testing/datatest_utils.py

```diff
@@ -122,7 +122,7 @@ def get_datapath_for_experiment_advection(ranked_base_path, experiment=REGIONAL_
 
 def create_icon_serial_data_provider(datapath, processor_props):
     # note: this needs to be here, otherwise spack doesn't find serialbox
-    from icon4py.model.common.test_utils.serialbox_utils import IconSerialDataProvider
+    from icon4py.model.testing.serialbox import IconSerialDataProvider
```

Comment:
can we not place this import outside? This way it doesn't have to be repeated in line 137